src/components/Inputs/WebRadio/Radio.vue

```diff
@@ -8,6 +8,7 @@
             :options="options"
             stacked
             plain
+            style="line-height: 2.5em"
```

Comment:
@djarecka to follow best practices, you could add the style to a class and use the class name in the html element. you can add this style to the class `text-left` here.

Reply:
could you please show me how exactly would you do it? Yesterday I've tried to change the `class` element and failed to see any difference, so must have done something wrong

Reply:
@djarecka last night when I used the commit checksum / commitish it worked, but now it doesn't so I'm guessing your latest change created a bug.

Reply:
@danielmlow - but i didn't change anything...

Reply:
@djarecka oh, sorry. It wasn't related to your commit. Disregard

Reply:
@djarecka @sanuann very minor, but I did notice that when `"multipleChoice": true` then there is more space for the reponse labels but it's a bit uneven with the boxes:
![image](https://user-images.githubusercontent.com/14787791/139594638-92378955-2edc-4b00-b390-2ee6c85776eb.png)

The `"multipleChoice": false` works fine:
![image](https://user-images.githubusercontent.com/14787791/139594671-3ab6a8dd-cafc-4afc-a379-2e497d530412.png)